src/prefect/server/api/ui/schemas.py

```diff
@@ -28,7 +28,7 @@ async def validate_obj(
     json_schema: dict[str, Any] = Body(
         ...,
         embed=True,
-        alias="schema",
+        validation_alias="schema",
```

Comment:
It's kinda wonky, but you could try setting both `validation_alias` and `serialization_alias`.
```suggestion
        validation_alias="schema",
        serialization_alias="schema",
```